modules/log6x-configuring-otlp-output.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output sends data over HTTP using JSON encoding by adhering to the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability Framework].
+
+:FeatureName: The OTLP output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* To enable OTLP, create or edit the `ClusterLogForwarder` CR using the template below:
++
+.Example ClusterLogForwarder CR
```

Comment:
@cahartma @anpingli @r2d2rnd Could you please confirm if we can use the following OTLP example? I will add callouts once we have consensus on OTLP example. 

Reference: https://gitlab.cee.redhat.com/aosqe/aosqe-tools/-/blob/master/logging/log_doc/clf-6.0-examples/otlp.yaml?ref_type=heads

Reply:
I can't say so much about the OTLP as not tested. But I'd suggest to don't put an example for each output that exists and I'll focus on

1. Be sure the API have all the details needed to configure the output
3. Add to the Table as it was done the new output
4. Make reference to `oc explain clusterLogForwarder.spec.outputs`  for Logging 5 and the respective command for Logging 6 to know the outputs and options supported for each output.

Reply:
I am commenting out this example as a placeholder.

@libander will cover this example as part of the larger PR (The larger PR, which will be created before Logging 6.0 release),